@here/harp-datasource-protocol/lib/DecodedTile.ts

```diff
@@ -217,6 +223,10 @@ export function getProjectionName(projection: Projection): string | never {
         return "webMercator";
     } else if (projection === sphereProjection) {
         return "sphere";
+    } else if (projection === equirectangularProjection) {
+        return "equirectangular";
+    } else if (projection === equirectangularWorldProjection) {
+        return "equirectangularworld";
```

Comment:
we use `camel case` for the projection names, so please rename it `equirectangularWorld`. Actually, about the name `equirectangularWorldProjection` it would be nice if we could fine a less ambiguous name.